js/index.js

```diff
@@ -1,37 +1,78 @@
 // ITERATION 1
-
 function updateSubtotal(product) {
-  console.log('Calculating subtotal, yey!');
+  const price = product.querySelector('.price span').innerText;
+  const quantity = product.querySelector('.quantity>input').value;
+  const subtotal = price * quantity
 
-  //... your code goes here
+  product.querySelector('.subtotal span').innerHTML = subtotal
+
+  return subtotal
 }
 
-function calculateAll() {
-  // code in the following two lines is added just for testing purposes.
-  // it runs when only iteration 1 is completed. at later point, it can be removed.
-  const singleProduct = document.querySelector('.product');
-  updateSubtotal(singleProduct);
-  // end of test
 
-  // ITERATION 2
-  //... your code goes here
+function calculateAll() {
+  const singleProduct = document.querySelector('.product')
```

Comment:
You don't need to update the subtotal for the singleProduct, because you do it below for all products